experiments.json

```diff
@@ -16,15 +16,6 @@
       "max": "100"
     }
   },
-  "search-term": {
-    "match": {
-      "appId": "^org.mozilla.fennec|^org.mozilla.firefox_beta$"
```

Comment:
This isn't "disabling" the experiment, but rather completely removing it. I suppose this will work out fine, since if the experiment is missing from the JSON, it will just act disabled for clients that still have the logic for the experiment.